include/z64.h

```diff
@@ -75,9 +75,9 @@ typedef struct {
 
 typedef struct {
     /* 0x0000 */ u32    size;
-    /* 0x0004 */ void*  bufp;
-    /* 0x0008 */ void*  head;
-    /* 0x000C */ void*  tail;
+    /* 0x0004 */ u8*    bufp;
+    /* 0x0008 */ u8*    head;
+    /* 0x000C */ u8*    tail;
```

Comment:
Is there a reason for this general change?

Reply:
I don't think so, must've slipped in by accident during some testing. I'll fix it soon